### PR TITLE
Add dbus tests into the source archive

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Related: rhbz#1508385

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

File: src/tests/Makefile.am [hunk 1]
```diff
@@ -7,6 +7,7 @@ NULL =
 EXTRA_DIST =                                                                   \
 	test_polkitd.py                                                        \
 	integration-test                                                       \
+	dbus-tests                                                             \
```
Comment (vpodzime) [inline src/tests/Makefile.am]: Is this really adding the directory with all its contents?

Comment (vojtechtrefny) [reply src/tests/Makefile.am]: Documentation says it does -- https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html -- and it really adds all files to the archive.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Pull request merged**